### PR TITLE
New website

### LINKED ISSUE #138: Versioned documentation
/area documentation

**What would you like to be added**:

We need to let the user navigate documentation for old versions.

**Why is this needed**:

Because not everyone runs on the latest version of Falco.

### PR DESCRIPTION
**Pull request by fntlnz**
Co-Authored-By: Leonardo Di Donato <leodidonato@gmail.com>
Co-Authored-By: Rajakavitha Kodhandapani <krajakavitha@gmail.com>
Co-Authored-By: Leonardo Grasso <me@leonardograsso.com>
Signed-off-by: Lorenzo Fontana <lo@linux.com>

For the advent of 0.27.0, @leodido and I thought that it's time for Falco to have a better website to help our users navigating the content in an easier way.

Goals of this PR:
- [x] to have everything in place, homepage, videos, etc
- [x] To have search
- [x] To have a more understandable typeface
- [x] To let users provide feedback (this is only enabled in the production build, preview here doesn't show)
- [x] To let users easily contribute/edit content
- [x] To have versioned documentation, so that on release 0f 0.27.0, you can still browse 0.26.0
- [x] To work on mobile browsers

Everyone is encouraged to help, this was brought up to discussion in the [community call](https://github.com/falcosecurity/community) of November 25th, 2020.

**Working group**: The next meeting of the Falco website Working group is on December 11th, more details [here](https://lists.cncf.io/g/cncf-falco-dev/viewevent?eventid=995445&calstart=2020-12-11)

Thanks for the help to:

@Rajakavitha1 
@leodido 
@leogr 

**Important** If you open the preview and you don't see anything, append `/docs` to the URL, we didn't style the home page yet.

**What type of PR is this?**

/kind cleanup

/kind design

/kind user-interface

**Any specific area of the project related to this PR?**

/area blog

/area documentation

/area videos

**What this PR does / why we need it**:

@leodido and I 

**Which issue(s) this PR fixes**:

Fixes #12
Fixes #138

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by fntlnz** [comment]
First preview here: https://deploy-preview-324--falcosecurity.netlify.app/docs/

**Comment by fntlnz** [comment]
The broken logo is due to the minification library used by hugo

https://github.com/gohugoio/hugo/issues/6475

This is why it breaks in the netlify preview but not locally with `make serve`

**Comment by fntlnz** [comment]
It requires `minify 2.7.3` to make hugo work with the svg logos while using `--minify`. They tried to update it but there are issues with it so I'll keep an eye. For now I'm removing minify at all. 

https://github.com/gohugoio/hugo/issues/7792

File: config.toml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+[services.googleAnalytics]
+# Comment out the next line to disable GA tracking. Also disables the feature described in [params.ui.feedback].
+id = "UA-00000000-0"
+
+# Additional menu items
+
+[[menu.main]]
+    name = "Documentation"
+    weight = 40
+    url = "https://falco.org/docs/"
```
Comment (fntlnz) [inline config.toml:123]: we probably want the relative url here to avoid issues in previews

File: config.toml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+[[menu.main]]
+    name = "Blog"
+    weight = 50
+    url = "https://falco.org/blog/"
```
Comment (fntlnz) [inline config.toml:127]: same here

File: config.toml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+# Language configuration
+
+
+# Everything below this are Site Params
+
+[params]
+copyright = "The Falco Authors"
+copyright_linux =  "Copyright © 2020 The Linux Foundation ®."
+privacy_policy = "https://policies.google.com/privacy"
```
Comment (fntlnz) [inline config.toml:139]: we need to find the CNCF url for this one

File: assets/scss/_styles_project.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+footer {
+    min-height: 200px;
```
Comment (leodido) [inline assets/scss/_styles_project.scss:2]: You always loved min-height! 🤣

File: config.toml [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+# Specify a value here if you have would like to reference a different branch for the other github settings
+# github_branch =
+
+time_format_blog = "Monday, January 02, 2006"
+time_format_default = "January 2, 2006"
+# Sections to publish in the main RSS feed.
+rss_sections = ["blog"]
+
+# Google Custom Search Engine ID. Remove or comment out to disable search.
+gcs_engine_id = "011217106833237091527:la2vtv2emlw"
```
Comment (leogr) [inline config.toml:190]: I have noticed that ID :point_up: comes from :point_right: https://github.com/google/docsy/blob/976ee59c947a153a5c4fe9adc620b32684505d73/userguide/config.toml#L129

@Rajakavitha1 I guess we need our own gcs_engine_id?

Or do we want to switch to offline search with Lunr.js?

cc @leodido @fntlnz

**Review by leogr — COMMENTED** [review]
Left a comment about the search feature. 
PTAL

File: config.toml (hunk 5 above)
Comment (leodido) [reply config.toml:190]: Yeah, that needs to be changed IMHO.

Comment (Rajakavitha1) [reply config.toml:190]: @leogr Yes, that is right, we need to change that. Using Lunr.js is a good option too. However, Google Engine might give us good SEO search options. @leodido  and @fntlnz Please share your thoughts.

Comment (fntlnz) [reply config.toml:190]: I think that Lunr is good however Google results seems more accurate as far as I can see on the Kubernetes website.

**Comment by fntlnz** [comment]
I'm rebasing this

**Comment by fntlnz** [comment]
I just added the featured video playlist in home page and localized it in all our supported languages.

- Can you take a look for Japanese in the preview here ( https://deploy-preview-324--falcosecurity.netlify.app/ja/) @oke-py @TakaoShimizu @KeisukeYamashita 
- Can you take a look for Korean in the preview here ( https://deploy-preview-324--falcosecurity.netlify.app/ko/) @bluefriday @futureCreator 
- Can you take a look for Chinese in the preview here (https://deploy-preview-324--falcosecurity.netlify.app/zh/ ) @nevermosby  

Thanks everyone :)

**Comment by fntlnz** [comment]
This is *almost* ready to go, we need to bring the logos to the home page

**Comment by nevermosby** [comment]
> I just added the featured video playlist in home page and localized it in all our supported languages.
> 
> * Can you take a look for Japanese in the preview here ( https://deploy-preview-324--falcosecurity.netlify.app/ja/) @oke-py @TakaoShimizu @KeisukeYamashita
> * Can you take a look for Korean in the preview here ( https://deploy-preview-324--falcosecurity.netlify.app/ko/) @bluefriday @futureCreator
> * Can you take a look for Chinese in the preview here (https://deploy-preview-324--falcosecurity.netlify.app/zh/ ) @nevermosby
> 
> Thanks everyone :)

It looks good in Chinese page. @fntlnz

**Review by leogr — DISMISSED** [review]
LGTM now, 

PS
anyway, needs a rebase

**Comment by TakaoShimizu** [comment]
I reviewed JP. I found some missing translation.

Falco, the cloud-native runtime security project, is the de facto Kubernetes threat detection engine
クラウドネイティブのランタイムセキュリティプロジェクトであるFalcoは、デファクトのKubernetes脅威検知エンジンです。

Falco was created by Sysdig in 2016 and is the first runtime security project to join CNCF as an incubation-level project. Falco detects unexpected application behavior and alerts on threats at runtime.
Falcoは2016年にSysdigによって作成され、CNCFにインキュベーションレベルのプロジェクトとして参加した初のランタイムセキュリティプロジェクトです。Falcoは、予期しないアプリケーションの動作を検出し、ランタイムで脅威を警告します。

Why Falco?
なぜFalco？

The flexible rules engine allows you to describe any type of host or container behavior or activity.
柔軟なルールエンジンにより、あらゆるタイプのホストやコンテナの動作やアクティビティを記述することができます。

End-Users
エンドユーザー

Vendors
ベンダー

Integrations
インテグレーション

**Comment by fntlnz** [comment]
Rebased and took care of the missing Japanese translation issues, thanks @TakaoShimizu !

**Review by leodido — DISMISSED** [review]
🔥

**Review by leodido — DISMISSED** [review]

**Review by leogr — APPROVED** [review]

**Review by leogr — DISMISSED** [review]

**Review by leodido — DISMISSED** [review]

**Review by leogr — APPROVED** [review]

**Pull request merged**